### ISSUE TITLE
Refresh button doesn't have screen reader label

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#2158]
Ideally, should add `aria-label` on `<button>`, but we don't have a mechanism for this yet. Found that putting it on the icon works with Voice Over at least.
